e2e/Dockerfile.node

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.9.2-alpine AS builder
+
+# Install packages
+RUN apk add --update bash g++ make\
+    && rm -rf /var/cache/apk/*
+
+WORKDIR /go/src/github.com/mysterium/node
+ADD . .
+RUN GOOS=linux GOARCH=amd64 bin/server_build && bin/client_build
+
+FROM alpine:3.6
+
+# Install packages
+RUN apk update \
+    && apk add --no-cache iptables ca-certificates openvpn \
+    && rm -rf /var/cache/apk/*
+
+WORKDIR /mysterium
+# Copy node stuff
+COPY --from=builder /go/src/github.com/mysterium/node/build/server/mysterium_server .
+COPY --from=builder /go/src/github.com/mysterium/node/bin/server_package/config server_config
+
+#Copy client stuff
+COPY --from=builder /go/src/github.com/mysterium/node/build/client/mysterium_client .
+COPY --from=builder /go/src/github.com/mysterium/node/bin/client_package/config client_config
+
+
+ENTRYPOINT ["/mysterium/mysterium_server"]
```

Comment:
You compile 2 executables, but use only `mysterium_server`. Do you have a case then You need run both executables *in same machine*?

Reply:
I am using the same image to start both server and node container - in that case container is built once, with two execs ready. See docker-compose node and client definitions.